test/src/kbasesearchengine/test/search/ElasticIndexingStorageTest.java

```diff
@@ -621,4 +625,129 @@ public void testTypeVersions() throws Exception {
         
     }
 
+    private void prepareTestMultiwordSearch(GUID guid1, GUID guid2, GUID guid3) throws Exception {
+        SearchObjectType objectType = new SearchObjectType("Simple", 1);
+        IndexingRules ir = IndexingRules.fromPath(new ObjectJsonPath("prop1"))
+                .withFullText().build();
+        List<IndexingRules> indexingRules= Arrays.asList(ir);
+
+            indexObject(guid1, objectType, "{\"prop1\":\"multiWordInSearchMethod1 multiWordInSearchMethod2\"}",
+                    "multiword.1", Instant.now(), null,
+                    true, indexingRules);
+            indexObject(guid2, objectType, "{\"prop1\":\"multiWordInSearchMethod2\"}",
+                    "multiword.2", Instant.now(), null,
+                    true, indexingRules);
+            indexObject(guid3, objectType, "{\"prop1\":\"multiWordInSearchMethod1\"}",
+                    "multiword.3", Instant.now(), null,
+                    true, indexingRules);
+    }
+
+
+    @Test
+    public void testMultiwordSearch() throws Exception{
+        GUID guid1 = new GUID("WS:11/1/2");
```

Comment:
Out of curiosity why are you using version 2? Doesn't matter, just wondering.

Reply:
b/c when I was starting to write the tests I was curious if it the database would let me. Got distracted with other things and forgot that I left it as version 2....